lib/cloud-watch-event-listener.js

```diff
@@ -147,39 +142,33 @@ class CloudWatchEventListener extends events.EventEmitter {
       } 
 
       if (dbResponse.lastEvent && dbResponse.lastEvent < generated) {
-        try {
```

Comment:
I'm confused why we can get rid of these try-catch now.

Reply:
Because I missed it while doing my pass through.  Thanks for catching it.

Reply:
Hmm, thinking about it more, we can probably get rid of the dbResponse.lastEvent checking here altogether and rely on the upsert's `ON CONFLICT... WHERE instances.terminated < $9` thingy.